sdk/monitor/Azure.Monitor.Query/tests/LogsTestData.cs

```diff
@@ -134,5 +136,26 @@ private async Task<int> QueryCount(string workspaceId)
                 return 0;
             }
         }
+
+        private async Task InitializeStorageAccount()
+        {
+            var client = new LogsQueryClient(_testEnvironment.Credential);
+            bool noData = true;
+
+            while (noData)
+            {
+                var result = await client.QueryResourceAsync(new ResourceIdentifier(_testEnvironment.StorageAccountId), "search *", new QueryTimeRange(DateTimeOffset.Now, TimeSpan.FromDays(15))).ConfigureAwait(false);
```

Comment:
Where does the 15 days come from? Is there a constant we can use for this if it is used elsewhere?

Reply:
I use the same range for checking if data is in the table, there is a datetime column that has the date set to 3, 7, and 10 days previously. It's a wider range to make sure I get everything. The storage account is created day of so maybe it doesn't need to be as wide?

Reply:
The account is created the same day for the pipeline, but we should use the same logic we use elsewhere to help speed up local testing. I'd suggest making the earliest threshold into a constant (10 days?) and use that in both places.